<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_nb_rangee_jeux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_jeu">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Jeu de cartes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jeu vidéo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jeu de table</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter_j">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modifier_jeux_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nb_joueur">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ajout_jeu">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout de jeux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_nb_joueur">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;nombre de joueur invalide.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_nb_rangee_jeux">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;le numéro de rangée est invalide.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nb_rangee_jeux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>nombre de rangée</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_createur">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer_jeux">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_code_jeux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;le code de jeux existe déjà.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre_jeux">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Titre du jeux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_createur">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>créateur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_code_jeux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>code de jeux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_createur">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;créateur invalide.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_jeux">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quitter_jeux">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_code_inv_jeux">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Le code de jeux est invalide.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code_jeux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_type_jeu">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>type de jeu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nb_joueur">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>nombre de joueur</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_jeux">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
